server/apps/publish/publish_filters/filter_condition.py

```diff
@@ -80,6 +80,18 @@ def on_update(self, updates, original):
         self._check_equals([doc])
         self._check_parameters([doc])
 
+    def delete(self, lookup):
+        referenced_filters = self._get_referenced_filter_conditions(lookup.get('_id'))
+        if referenced_filters.count() > 0:
+            references = ','.join([pf['name'] for pf in referenced_filters])
```

Comment:
**A tip:** in cases like this when an iterable is expected as an argument, there is no need to construct and pass in a (temporary) list, a generator expression achieves the same with less overhead.

(I know I'm late to the party with this PR already merged, but it might be useful in the future)